test/cpp/end2end/client_interceptors_end2end_test.cc

```diff
@@ -19,17 +19,21 @@
 #include <memory>
 #include <vector>
 
+#include "absl/memory/memory.h"
```

Comment:
Any reason why this header order changed?

Reply:
Made the change while I was at it (following the google C++ style guide on header ordering)